modules/ossm-migrating-complete-multitant-cert-manager.adoc

```diff
@@ -0,0 +1,28 @@
+
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/multitenant/ossm-migrating-multitenant-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="migrating-complete-multitant-cert-manager_{context}""]
+== Completing a multitenant deployment with cert-manager migrating
+
+.Prerequisites
+
+* You have migrated a multitenant deployment with the cert-manager and istio-csr tools.
+
+.Procedure
+
+. Verify that your new injection label is present in all workload namespaces.
+
+. Update the `app.controller.configmapNamespaceSelector` field by running the following command:
++
+[source,terminal]
+----
+helm upgrade cert-manager-istio-csr jetstack/cert-manager-istio-csr \
+   --install \
+   --reuse-values \
+   --namespace istio-system \
+   --wait \
+   --set "app.controller.configmapNamespaceSelector=istio-injection=enabled"
```

Comment:
This is not the right label for `MultiTenant`. It should match what you specify for your `discoverySelectors`. In the MultiTenant examples this is: `tenant: tenant-a`
```suggestion
   --set "app.controller.configmapNamespaceSelector=tenant=tenant-a
```